README.md

```diff
@@ -36,29 +35,37 @@ npm install @dashevo/dapi-client
 const DAPIClient = require('@dashevo/dapi-client');
 var client = new DAPIClient();
 
-client.getBalance('testaddress');
+client.getBestBlockHash().then((hash) => {
+  console.log(hash);
+});
 ```
 
 ### Custom seed node
 
-Custom seed nodes are necessary for connecting the client to devnets since the
-client library is unaware of them otherwise.
+Custom seed nodes are necessary for connecting the client to devnets since the client library is unaware of them otherwise.
 
 ```javascript
 const DAPIClient = require('@dashevo/dapi-client');
 
 var client = new DAPIClient({
   seeds: [{
-    service: 'example.com:9999',
+    service: 'seed-1.evonet.networks.dash.org',
     port: 3000
   }],
 });
 
-var blockHeight = client.getBestBlockHeight();
+client.getBestBlockHash().then((r) => {
+  console.log(r);
+});
+
 ```
 
```

Comment:
I would add one more section on how to specify specific node to make request (without DML) with addresses option.

Also, I think we should mention that now you can pass all DAPI Client options directly to any command to have specific behavior for that call.

Reply:
Why don't we merge these changes and a separate PR for additional updates?

Reply:
Sounds good!